infra/terraform/hash/main.tf

```diff
@@ -298,10 +298,6 @@ module "application" {
   ])
   api_image              = module.api_ecr
   api_migration_env_vars = concat(var.hash_api_migration_env_vars, [
-    {
-      name  = "INCOMPLETE_USER_ACCOUNTS", secret = true,
-      value = sensitive(data.vault_kv_secret_v2.secrets.data["incomplete_user_accounts"])
-    },
   ])
```

Comment:
```suggestion
  api_migration_env_vars = var.hash_api_migration_env_vars
```

Reply:
This is the pattern we have for other env vars around it and is what it was previously, does it serve no purpose?

Reply:
We probably had it because at some point we had inline variables as well. For empty arrays, it doesn't serve any purpose, no. 🙂